adal/src/main/java/com/microsoft/aad/adal/WebRequestHandler.java

```diff
@@ -54,6 +54,10 @@
 
     private UUID mRequestCorrelationId = null;
 
+    private String mCurrentAdalClientVersion = AuthenticationContext.getVersionName();
+
+    private String mBrokerVersion = null;
```

Comment:
this is a variable I added to make it both work for broker and public ADAL. In broker, we need to add the broker version as a parameter in header. But it's not necessary in public ADAL, so I introduced this variable, if it's not set (in public ADAL) then the broker version won't be passed as a parameter in header.